perfect-square/perfect.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	fmt.Println("The program determines if an integer is a perfect square.")
+	fmt.Println("Enter the number:")
+	var n int
+	i := 1
+	a := 1
+	fmt.Scan(&n)
+	for i <= n {
+		i++
+		if i*i == n {
+			a++
```

Comment:
Instead of incrementing a (`a++`) and then printing,  just print and return, i.e.

```
fmt.Println(n, "is a perfect square")
return
```